ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/cveListExceptionFlow.test.js

```diff
@@ -67,24 +73,51 @@ describe('Workload CVE List deferral and false positive flows', () => {
     it('should defer a single CVE', () => {
         visitWorkloadCveOverview();
 
-        selectSingleCveForException('DEFERRAL');
+        selectSingleCveForException('DEFERRAL').then((cveName) => {
+            verifySelectedCvesInModal([cveName]);
+            fillAndSubmitExceptionForm('Test comment', 'When all CVEs are fixable');
```

Comment:
What do you think about using an object as a parameter for this function? I think it would make it easier to determine what these values translate to. For example:

```
fillAndSubmitExceptionForm({
    comment: "Test comment",
    scopeLabel: "When all CVEs are fixable"
});
```

Reply:
100% agree. I actually added this in the follow up to this PR since there is yet another parameter that is needed.